libia2/include/ia2.h

```diff
@@ -116,8 +107,8 @@ static uint32_t ia2_get_pkru() {
 /// \param data pointer to a PhdrSearchArgs structure
 ///
 /// The callback passed to dl_iterate_phdr in the constructor inserted by
-/// INIT_COMPARTMENT to pkey_mprotect the pages corresponding to the
-/// compartment's loaded segments.
+/// ia2_init.c to pkey_mprotect the pages corresponding to the compartment's
```

Comment:
Slight bikeshed but I think `ia2_init_compartment.inc` would be a better name for the file--`.c` files should be standalone-compileable, and it doesn't initialize ia2 as a whole but only one compartment.